toxcore/tox.h

```diff
@@ -4736,6 +4736,11 @@ typedef enum Tox_Err_Group_Send_Custom_Packet {
      */
     TOX_ERR_GROUP_SEND_CUSTOM_PACKET_DISCONNECTED,
 
+    /**
+     * The packet failed to send.
```

Comment:
maybe be more descriptive here. this does not differentiate between `any` and `all` .

Reply:
or do you have further plans for this ?

Reply:
What sort of plans?

Reply:
*shrug*